lib/statsample/version.rb

```diff
@@ -1,3 +1,3 @@
 module Statsample
-  VERSION = '2.0.1'
+  VERSION = '2.0.2'
```

Comment:
Don't bump the version. It will go in a separate commit. Just remove this part and I'll merge.